src/DynamoCoreWpf/Views/GuidedTour/WebBrowserWindow.xaml.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.IO;
+using System.Windows;
+using System.Windows.Controls.Primitives;
+using System.Windows.Media;
+using Dynamo.Wpf.UI.GuidedTour;
+using Dynamo.Wpf.ViewModels.GuidedTour;
+
+namespace Dynamo.Wpf.Views.GuidedTour
+{
+    /// <summary>
+    /// Interaction logic for PopupWindow.xaml
+    /// </summary>
+    public partial class WebBrowserWindow : Popup
+    {
+        //This variable represents a sum of the header and footer of the main popup to compensate the hight of this popup
+        private const int headerAndFooterCompensation = 150;
+        //This variable represents the width size of the tooltip to relocate this popup
+        private const int tooltipOffset = 10;
+        //The headers size of the main popup to add an offset and adjust this popup
+        private const int headerOffset = 50;
+        //Directory where the html files are located
+        private const string htmlFileDirectory = "file:///{0}/Views/GuidedTour/HtmlPages/{1}.html";
```

Comment:
are these .html files really loaded from a directory called "Views" in distributed builds of Dynamo? I think you should double check how localization was performed for the existing localized .html files that show in the docs browser @QilongTang @filipeotero - because I believe the localization team wanted the html filed to be packaged into the resource binary.

Reply:
Good point @mjkkirschner Team were just talking about how to localize this html, do you recall if localization team support whole sweep of them or the strings still need to be resource strings?

Reply:
@QilongTang I think they do the whole html file - but their tools run better on .html files embedded into the resource binaries. Please checkout the docs browser project for example. I think we might also need to contact them and let them know theres html files embedded into other binaries for them to consume.

Reply:
this also implies that to load the html content you'll want to use the name of the file, with the resource manager for the binary to pull out the correctly localized version.

Reply:
@mjkkirschner Thanks! These comments helps a lot. @filipeotero do you mind follow this example to set up the html content and the corresponding path?
![image](https://user-images.githubusercontent.com/3942418/135492228-dd124768-4555-4c3d-9b54-d90f1476fa2e.png)

Reply:
Sure, I will load the HTML files from the binaries now. Thanks, @QilongTang @mjkkirschner.